Missed README auto generation for hddldaemon image

When run below command:

- cd build
- cmake cmake -DUPDATE_IMAGE_README=ON ..
- make

Below error shown on screen:
Scanning dependencies of target build_vcaca
Traceback (most recent call last):
  File "/home/vcse/ovc/Dockerfiles/script/generate_readme.py", line 306, in <module>
    main()
  File "/home/vcse/ovc/Dockerfiles/script/generate_readme.py", line 303, in main
    walk_path(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
  File "/home/vcse/ovc/Dockerfiles/script/generate_readme.py", line 130, in walk_path
    create_readme(root, path_components)
  File "/home/vcse/ovc/Dockerfiles/script/generate_readme.py", line 298, in create_readme
    my_file.write(quick_reference(path, image_name, image_type, image_os, image_platform))
  File "/home/vcse/ovc/Dockerfiles/script/generate_readme.py", line 161, in quick_reference
    text_holder += included_components(image_name)
  File "/home/vcse/ovc/Dockerfiles/script/generate_readme.py", line 149, in included_components
    for comp in included_subs[image_name]:
KeyError: 'hddldaemon'
VCAC-A/CMakeFiles/build_vcaca.dir/build.make:57: recipe for target 'VCAC-A/CMakeFiles/build_vcaca' failed
make[2]: *** [VCAC-A/CMakeFiles/build_vcaca] Error 1
CMakeFiles/Makefile2:90: recipe for target 'VCAC-A/CMakeFiles/build_vcaca.dir/all' failed
make[1]: *** [VCAC-A/CMakeFiles/build_vcaca.dir/all] Error 2
Makefile:94: recipe for target 'all' failed
make: *** [all] Error 2

--- a/VCAC-A/ubuntu-18.04/analytics/hddldaemon/README.md
+++ b/VCAC-A/ubuntu-18.04/analytics/hddldaemon/README.md
@@ -1,0 +1,39 @@
+This docker image is part of Open Visual Cloud software stacks. Optimized for Media Analytics. 
+
+## Supported tags and respective Dockerfile links
+ - [vcac-a-ubuntu-1804-analytics-hddldaemon](https://github.com/OpenVisualCloud/Dockerfiles/blob/master/VCAC-A/ubuntu-18.04/analytics/hddldaemon/Dockerfile)
+
+## Quick reference
+- #### Supported platform and OS
+Intel&reg; VCAC-A platform, Ubuntu 18.04
+
+
+
+
+- #### Where to get help:
+- [Open Visual Cloud Dockerfiles Github](https://github.com/OpenVisualCloud/Dockerfiles)
+- [Getting Started With Open Visual Cloud Docker Files](https://01.org/openvisualcloud/documents/get-started-docker)
+- [the Docker Community Forums](https://forums.docker.com)
+- [the Docker Community Slack](https://www.docker.com/docker-community)
+- [Stack Overflow](https://stackoverflow.com/questions/tagged/docker)
+
+-  #### Where to file issues:
+[OpenVisualCloud Dockerfiles Issues](https://github.com/OpenVisualCloud/Dockerfiles/issues)
+
+- #### Maintained by:
+[OpenVisualCloud Dockerfiles Community](https://github.com/OpenVisualCloud/Dockerfiles/graphs/contributors)
+
+
+## License
+This docker installs third party components licensed under various open source licenses.  The terms under which those components may be used and distributed can be found with the license document that is provided with those components.  Please familiarize yourself with those terms to ensure your distribution of those components complies with the terms of those licenses.
+
+
+| Components | License |
+| ----- | ----- |
+|Ubuntu| [Various](https://hub.docker.com/_/ubuntu) |
+|libusb|GNU Lesser General Public License v2.1|
+|OpenVINO|End User License Agreement for the Intel(R) Software Development Products|
+
+
+More license information can be found in [components source package](https://github.com/OpenVisualCloud/Dockerfiles-Resources).   
+As for any pre-built image usage, it is the image user's responsibility to ensure that any use of this image complies with any relevant licenses and potential fees for all software contained within. We will have no indemnity or warranty coverage from suppliers.

--- a/script/generate_readme.py
+++ b/script/generate_readme.py
@@ -115,11 +115,12 @@ license_subs = {
                 "svt-hevc.1-3-0" : ["|Intel SVT-HEVC|BSD-2-Clause Plus Patent License|"],
                 "svt-hevc" : ["|Intel SVT-HEVC|BSD-2-Clause Plus Patent License|"],
                 "svt-vp9" : ["|Intel SVT-VP9|BSD-2-Clause Plus Patent License|"],
-                "usrsctp" : ["|usrsctp|BSD 3-clause \"New\" or \"Revised\" License|"]
+                "usrsctp" : ["|usrsctp|BSD 3-clause \"New\" or \"Revised\" License|"],
+                "libusb" : ["|libusb|GNU Lesser General Public License v2.1|"]
                }
 
 # M4 files for which no license is needed
-license_exclude = ['automake', 'build-tools', 'cleanup', 'cmake', 'install', 'install.pkgs', 'install.pkgs.owt', 'libfdk-aac', 'libmp3lame', 'nasm', 'nginx-cert', 'nginx-conf', 'qat', 'transform360', 'yasm', 'libva-utils', 'ospray-example_san-miguel', 'ospray-example_xfrog']
+license_exclude = ['automake', 'build-tools', 'build-tools-hddl', 'build-tools-hddl-layer', 'cleanup', 'cmake', 'install', 'install.pkgs', 'install.pkgs.owt', 'libfdk-aac', 'libmp3lame', 'nasm', 'nginx-cert', 'nginx-conf', 'qat', 'transform360', 'yasm', 'libva-utils', 'ospray-example_san-miguel', 'ospray-example_xfrog']
 
 # Walk through the repo and find folder with Dockerfiles.m4
 def walk_path(path):
@@ -146,9 +147,12 @@ def url_generator(local_path, image_name, image_type, image_os, image_platform):
 # Generate links to docs of included components
 def included_components(image_name):
     included_holder = ''
-    for comp in included_subs[image_name]:
-        included_holder += comp
-        included_holder += '\t'
+    if image_name in included_subs:
+        included_holder += "- #### Included components:\n"
+        for comp in included_subs[image_name]:
+            included_holder += comp
+            included_holder += '\t'
+    included_holder += '\n\n'
     return included_holder
     
 # Generate quick reference part of README
@@ -157,9 +161,7 @@ def quick_reference(local_path, image_name, image_type, image_os, image_platform
     text_holder += "- #### Supported platform and OS\n"
     text_holder += "Intel&reg; "+platform_subs[image_platform]+", "+os_subs[image_os]
     text_holder += "\n\n"
-    text_holder += "- #### Included components:\n"
     text_holder += included_components(image_name)
-    text_holder += "\n\n"
     text_holder +="""
 - #### Where to get help:
 - [Open Visual Cloud Dockerfiles Github](https://github.com/OpenVisualCloud/Dockerfiles)

